core/Cargo.toml

```diff
@@ -12,7 +12,7 @@ byteorder = "1.4"
 bitstream-io = "1.6.0"
 flate2 = "1.0.25"
 fnv = "1.0.7"
-gc-arena = { git = "https://github.com/ruffle-rs/gc-arena" }
+gc-arena = { git = "https://github.com/ruffle-rs/gc-arena", rev = "3195fbed9f2ba1683fb113e3d7d84794fe3465d5" }
```

Comment:
Am I missing something or shouldn't this be kyren/gc-arena now?

Reply:
You're right - we no longer need our fork.

Reply:
Oops, right, good catch :c